packages/node-integration-tests/suites/express/tracing/server.ts

```diff
@@ -0,0 +1,26 @@
+import * as Sentry from '@sentry/node';
+import * as Tracing from '@sentry/tracing';
+import cors from 'cors';
+import express from 'express';
+
+const app = express();
+
+Sentry.init({
+  dsn: 'https://public@dsn.ingest.sentry.io/1337',
+  release: '1.0',
+  integrations: [new Sentry.Integrations.Http({ tracing: true }), new Tracing.Integrations.Express({ app })],
```

Comment:
Can we also check if outgoing requests get the sentry trace header attached? Would also like to see the opposite, an incoming request get a transaction created based on the incoming sentry-trace header.

Reply:
@AbhiPrasad, I added the tests. Kind of hacky though.

Reply:
This is fine as a starting point, we can always improve afterwards